src/requires.jl

```diff
@@ -15,40 +15,38 @@ function __init__()
     end)
 
     @require Colors="5ae59095-9a9b-59fe-a467-6f913c188581" begin
-        fixnotation(c::Float64) = round(c, digits = 4)
+        function PGFPlotsX._rgb_for_printing(c::Colors.Colorant)
+            rgb = convert(Colors.RGB{Float64}, c)
+            round.((red(rgb), green(rgb), blue(rgb)); digits = 4) # somewhat arbitrary choice of accuracy
+        end
+        
         function PGFPlotsX.print_opt(io::IO, c::Colors.Colorant)
-            rgb = convert(Colors.RGB, c)
-            _rgb_64 = convert.(Float64, (rgb.r, rgb.g, rgb.b))
-            rgb_64 = fixnotation.(_rgb_64)
+            rgb_64 = _rgb_for_printing(c)
             print(io, "rgb,1:",
                   "red,"  , rgb_64[1], ";",
                   "green,", rgb_64[2], ";",
                   "blue," , rgb_64[3])
         end
-
+    
         function PGFPlotsX.print_tex(io::IO, c::Tuple{String, Colors.Colorant}, ::Any)
             name, color = c
-            rgb = convert(Colors.RGB, color)
-            _rgb_64 = convert.(Float64, (rgb.r, rgb.g, rgb.b))
-            rgb_64 = fixnotation.(_rgb_64)
+            rgb_64 = _rgb_for_printing(color)
             print(io, "\\definecolor{$name}{rgb}{$(rgb_64[1]), $(rgb_64[2]), $(rgb_64[3])}")
         end
-
+    
         function PGFPlotsX.print_tex(io::IO,
-                                     c::Tuple{String, Vector{<:Colors.Colorant}},
-                                     ::Any)
+            c::Tuple{String, Vector{<:Colors.Colorant}},
```

Comment:
why did you change the indentation? this is pretty non-standard, please revert.

Reply:
The overarching problem I have is that I'm editing things in the browser by copying the code to vim, auto indenting everything, fixing it all, and pasting it back. This is horrible. In the old julia there was a nice work flow (written by Tim Holy I think) on how to contribute to existing packages -- that's gone now, or at least I can't find it in the docs. How do you do it?

Reply:
See [eg this description](https://discourse.julialang.org/t/recommended-way-to-develop-fix-with-packages/14436/2).

Reply:
Awesome. Thank you.